classes/db/Db.php

```diff
@@ -553,7 +553,8 @@ public function execute($sql, $use_cache = true)
 	 * @param string|DbQuery $sql Query to execute
 	 * @param bool $array Return an array instead of a result object (deprecated since 1.5.0.1, use query method instead)
 	 * @param bool $use_cache
-	 * @return array|false|null|mysqli_result|PDOStatement|resource
+	 *
+	 * @return array|false
```

Comment:
Hi, thank you for your contribution,

but executeS can return the result of query call, and in PDO is PDOStatement.

Best regards